sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/implementation/ReflectionSerializable.java

```diff
@@ -26,13 +26,16 @@
  */
 public final class ReflectionSerializable {
     private static final ClientLogger LOGGER = new ClientLogger(ReflectionSerializable.class);
-    private static final Map<Class<?>, ReflectiveInvoker> FROM_JSON_CACHE = new ConcurrentHashMap<>();
+    private static final Map<Class<?>, ReflectiveInvoker> FROM_JSON_CACHE;
+
+    static {
+        FROM_JSON_CACHE = new ConcurrentHashMap<>();
```

Comment:
What's the difference between instantiating this here and at the constant declaration?